spec/factories/new_registration.rb

```diff
@@ -27,28 +27,29 @@
     end
 
     trait :complete do
+      limited_company
+
       location { "england" }
-      applicant_first_name { "Joe" }
-      applicant_last_name { "Bloggs" }
+      applicant_first_name { Faker::Name.first_name }
+      applicant_last_name { Faker::Name.last_name }
       applicant_phone { "01234567890" }
-      applicant_email { "test@example.com" }
-      business_type { "limitedCompany" }
+      applicant_email { Faker::Internet.email }
```

Comment:
Just to be extra, extra careful when we have used faker to generate emails in the past, we have used `Faker::Internet.safe_email` as it will generate **[name]@example.[org|com|net]**.

This is just to ensure we don't inadvertently generate a valid email address and then mistakenly send it.

```suggestion
      applicant_email { Faker::Internet.safe_email }
```

Reply:
Nice one! 👍